R/ROCmLLVM/common.jl

```diff
@@ -0,0 +1,52 @@
+const ROCM_GIT = "https://github.com/RadeonOpenCompute/llvm-project"
+const ROCM_TAGS = Dict(
+    v"4.2.0" => "b204d7f0cae65b6cd4446eec50fc1fb675d582af",
+    v"4.5.2" => "9bbd96fd1936641cd47defd8022edafd063019d5",
+)
+const LLVM_VERSIONS = Dict(
+    v"4.2.0" => v"12.0.0",
+    v"4.5.2" => v"13.0.0",
+)
+const ROCM_PLATFORMS = [
+    Platform("x86_64", "linux"; libc="glibc", cxxstring_abi="cxx11"),
+    Platform("x86_64", "linux"; libc="musl", cxxstring_abi="cxx11"),
+]
+
+function configure_build(rocm_version; assert::Bool = false)
+    isfile("../buildscript.sh") || error("Cannot find `./buildscript.sh`!")
+    buildscript = read("../buildscript.sh", String)
+
+    git_version = ROCM_TAGS[rocm_version]
+    llvm_version = LLVM_VERSIONS[rocm_version]
+
+    sources = [
+        GitSource(ROCM_GIT, git_version),
+        DirectorySource("./bundled"),
+    ]
+
+    llib = "llvm/lib"
+    lbin = "llvm/bin"
+    products = [
```

Comment:
What about the LLVM static libraries? It would be good to ensure that they're built, for downstream dependencies.

Reply:
Added a small subset of them. I guess there's no need to list them all?

Reply:
Yeah, sounds good to me!